### ISSUE TITLE
Splitting out DB to separate container?

### ISSUE THREAD

**Issue by EricJoGonzalez** [MISP/x_old_misp_docker#85]
Is there any reason why the DB can't be split in to a separate container?

**Comment by namishelex01** [comment]
Yes, you can. Create a separate container and add misp image dependent upon db image in the docker compose configuration file.

**Comment by legoguy1000** [comment]
With the most recent commit, you are now installing MariaDB into the image.  Why? The whole point of containers is to minimize the amount of services in a single image.

**Comment by TheOfficalNick** [comment]
Hi @EricJoGonzalez. Have you successfully done this yet? I am looking to do something similar and came across this issue.

**Comment by legoguy1000** [comment]
Our custom docker image years a separate redis and db container. Works really well

**Comment by ktb-jcm** [comment]
Also See: https://github.com/RH-ISAC/misp-docker
